### DISCUSSION TITLE
Change header and footer colors

### DISCUSSION THREAD

**Discussion by lpinilla** [marp-team/marp#363] [Q&A] [answered]
Hi!

Is there any local directive to change the color of the header and footer? I know it can be done via:

```
<style>
  header,footer {
    color: red;
  }
</style>
```
but this will be applied to the whole doc and I'm looking for changing it on some specific slides.

Thank you!

**Comment by yhatt** [comment, marked as answer]
> I'm looking for changing it on some specific slides.

https://marpit.marp.app/theme-css?id=scoped-style

### Related

- #337

**Reply by lpinilla** [reply]
Thanks for always replying so fast!
